usr.bin/migcom/Makefile

```diff
@@ -29,18 +29,18 @@ SRCS= ident.c error.c global.c header.c \
 	statement.c string.c type.c user.c utils.c
 
 
-CFLAGS+= -I${.CURDIR} -I${.CURDIR}/shims -I${.CURDIR}/../../include
+CFLAGS+= -I${.CURDIR} -I${.CURDIR}/shims -I${.CURDIR}/../../include -Iinclude
```

Comment:
This should probably be `-I{.OBJDIR}/include`. Or get rid of `<stdio.h>` completely.